packages/react-components/src/index.css

```diff
@@ -1,16 +1,16 @@
-:root {
-    --o-ui-disabled-opacity: 0.4;
-    --o-ui-pill-border-radius: 999px;
-    --o-ui-rounded-border-radius: var(--scale-alpha);
-    --o-ui-circular-border-radius: 100%;
-}
-
 [class^="o-ui-"] {
     --o-ui-focus-ring-inset-sm: 0 0 0 1px var(--o-ui-focus-ring-color);
     --o-ui-focus-ring-inset-md: 0 0 0 2px var(--o-ui-focus-ring-color);
     --o-ui-focus-ring-inset-lg: 0 0 0 4px var(--o-ui-focus-ring-color);
     --o-ui-focus-ring: 0 0 0 2px var(--white), 0 0 0 4px var(--o-ui-focus-ring-color);
+    --o-ui-focus-ring-offset: 2px;
+    --o-ui-focus-ring-inner-offset: 1px;
```

Comment:
`inner-offset` doesn't make much sense to me.

Reply:
this is opposed to the outer offset, how would you name this variable since it's use is only to "insert" the focus ring in the element.

Reply:
Make sense, but how was I supposed to understand this by looking at these var names?

`--o-ui-focus-ring-offset: 2px;` should somehow include outer then

Reply:
totall;